test/suites/docker-compose.override.yml

```diff
@@ -40,5 +40,19 @@ services:
       WIKIBASE_PUBLIC_URL: http://${WIKIBASE_PUBLIC_HOST}
       QUICKSTATEMENTS_PUBLIC_URL: http://${QUICKSTATEMENTS_PUBLIC_HOST}
 
+  browser:
+    image: ${TEST_BROWSER_IMAGE_URL}
+    stop_grace_period: 1s
+    ports:
+      - 4444:4444
+      - 5900:5900
+      - 7900:7900
+    volumes:
+      - .:/usr/src/test
+    shm_size: 2g
+    environment:
+      SE_NODE_MAX_SESSIONS: ${MAX_INSTANCES}
+      SE_VNC_NO_PASSWORD: 1
+
```

Comment:
Can you help me understand the restructuring here?

Reply:
The `wbs-dev-runner` form the root `Dockerfile` / `docker-compose.yml` is now our general purpose  container as test-runner had become. It is what is always ran whenever we run any `./nx <lint|test|build|release>` commands (so pretty much everything). Things are structured such with `TestEnv`  that this set of services is what is booted before each test suite is ran, which is when we also need the browser available. I think it ultimately fits a bit better here than anywhere else remaining, but it also was a needed move so the browser service wasn't needing to be managed along with he dev-runner... If that makes sense. It may still shift around a bit, but this is a good spot for it for the moment.